src/main/java/org/testng/TestRunner.java

```diff
@@ -295,6 +293,22 @@ private void init(
     init();
   }
 
+  /**
+   * Returns all packages to use for the current test. This includes the test from the test
+   * suite. Never returns null.
+   */
+  private List<XmlPackage> getAllPackages()
+  {
+    final List<XmlPackage> allPackages = Lists.newArrayList();
+    final List<XmlPackage> suitePackages = this.m_xmlTest.getSuite().getPackages();
+    if (suitePackages != null)
```

Comment:
The `null` check is not needed, because `org.testng.xml.XmlSuite#getPackages` returns an empty list always.

Reply:
Ok, then lets leave the `null` checks as is. But please help address the styling discrepancy.